includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -193,7 +193,8 @@ private function createApiClient() {
 		// cURL function is allowed to execute not longer than 1 second
 		$api->getApiClient()
 				->getConfig()
-				->setCurlTimeout(1);
+				//todo bring back the default value (1)
```

Comment:
todo!

Reply:
@hakubo we'll do it as soon as we'll be ready to merge this code, since it timeouts on devboxes. It happens because dev master db is now in reston and when infobox template is created, then template-classification-storage service from POZ needs to execute several queries/inserts on master db in RES. We should make some environment-based switch to fix it, but I think it is topic for separate PR